data/entity/npc/monster/ice_giant/ice_giant.npcs.toml

```diff
@@ -0,0 +1,30 @@
+[ice_giant]
+id = 111
+hitpoints = 700
+att = 40
+str = 40
+def = 40
+max_hit_melee = 7
+attack_speed = 4
+hunt_mode = "aggressive"
+style = "slash"
+respawn_delay = 30
+categories = ["ice_giant"]
+drop_table = "ice_giant.drops"
```

Comment:
```suggestion
```
Not necessary as it'll use the category, if you did need it then it would be `drop_table = "ice_giant"`, no `.drops` required